From e33dd254a035363c122231aa11e931d8135d9998 Mon Sep 17 00:00:00 2001
From: Sascha Hauer <s.hauer@pengutronix.de>
Date: Tue, 10 Mar 2015 16:41:03 +0100
Subject: [PATCH 22/27] FROMLIST: ARM64: MediaTek MT8173: Add SCPSYS device
 node

This adds the SCPSYS device node to the MT8173 dtsi file.

Change-Id: I8207881a2dc04b4570208a8ee57f1be44136b760
Pathwork: https://patchwork.kernel.org/patch/5978601/
Signed-off-by: Sascha Hauer <s.hauer@pengutronix.de>
Signed-off-by: Eddie Huang <eddie.huang@mediatek.com>
---
 arch/arm64/boot/dts/mediatek/mt8173.dtsi | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/mediatek/mt8173.dtsi b/arch/arm64/boot/dts/mediatek/mt8173.dtsi
index d2aed57..1481e1f 100644
--- a/arch/arm64/boot/dts/mediatek/mt8173.dtsi
+++ b/arch/arm64/boot/dts/mediatek/mt8173.dtsi
@@ -146,10 +146,10 @@
 			#reset-cells = <1>;
 		};
 
-		scpsys: scpsys@10001000 {
+		scpsys: scpsys@10006000 {
 			compatible = "mediatek,mt8173-scpsys";
-			reg = <0 0x10001000 0 0x1000>, <0 0x10006000 0 0x1000>, <0 0x13fff000 0 0x1000>;
-			#clock-cells = <1>;
+			#power-domain-cells = <1>;
+			reg = <0 0x10006000 0 0x1000>;
 		};
 
 		pericfg: pericfg@10003000 {
-- 
1.9.1

